build-system/tasks/report-test-status.js

```diff
@@ -70,15 +71,16 @@ function inferTestType() {
 }
 
 async function postReport(type, action) {
-  if (type !== null && isTravisPullRequestBuild()) {
+  if (type !== null && isTravisBuild() && isPullRequestBuild()) {
```

Comment:
Can `isTravisBuild()` be removed here?

Reply:
Good question. Not until the internals of test reporting have been updated to work outside Travis. I've added a TODO.